perllib/FixMyStreet/Cobrand/Bexley.pm

```diff
@@ -365,5 +365,8 @@ sub dashboard_export_problems_add_columns {
     });
 }
 
+# Waste
+
+sub bin_day_format { '%A, %-d~~~ %B %Y' }
```

Comment:
I think it might make sense to put this in `perllib/FixMyStreet/Cobrand/Bexley/Waste.pm`, with the other WasteWorks related code, what do you think?

Reply:
Yep! Think I got muddled because Peterborough have everything in one file.